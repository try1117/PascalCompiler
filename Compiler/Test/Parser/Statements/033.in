program test;
var
  recArr: array[1..5] of record
    a: array['a'..'z'] of array[-10..10] of string;
    d: double;
  end;
  i: integer;
begin
  for i := 1 to 5 do begin
    write(recArr[i]);
  end;
end.
