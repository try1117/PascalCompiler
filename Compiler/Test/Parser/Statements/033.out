Lexical exception in position (10,5) - Can't read or write variables of type Record
   a : Array [97, 122] of Array [-10, 10] of String
   d : Double
end
